yt/data_objects/derived_quantities.py

```diff
@@ -237,33 +243,35 @@ class CenterOfMass(DerivedQuantity):
     >>> print ad.quantities.center_of_mass()
 
     """
-    def count_values(self, use_gas = True, use_particles = False):
+    def count_values(self, use_gas = True, use_particles = False, particle_type="all"):
+        if not particle_type in self.data_source.ds.particle_types:
```

Comment:
`flake8` would prefer this to be phrased `if particle_type not in self.data_source.ds.particle_types`. This also happens a few more times below.